ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Degenerate.hs

```diff
@@ -373,13 +387,14 @@ instance NoHardForks b => CommonProtocolParams (DegenFork b) where
 instance NoHardForks b => CanForge (DegenFork b) where
   type ExtraForgeState (DegenFork b) = ExtraForgeState (HardForkBlock '[b])
 
-  forgeBlock cfg forgeState block (Ticked slot (DLgr lgr)) txs proof =
+  forgeBlock cfg forgeState bno sno (TDLgr lgr) txs proof =
       DBlk . inject' (Proxy @(I b)) $
         forgeBlock
           (projCfg cfg)
           (project (castForgeState forgeState))
-          block
-          (Ticked slot (project lgr))
+          bno
+          sno
+          (unComp . project . Comp $ lgr) -- (Ticked slot (project lgr))
```

Comment:
Stray comment

Reply:
Turns out this too was delegating to `b` rather than the HFC. Fixed:

```haskell
  forgeBlock cfg forgeState bno sno (TDLgr lgr) txs proof = DBlk $
      forgeBlock
        (castTopLevelConfig cfg)
        (castForgeState forgeState)
        bno
        sno
        lgr
        (map unDTx txs)
        proof
```